.github/temp/test.yml

```diff
@@ -0,0 +1,33 @@
+name: Test
```

Comment:
@putmantime Here's the test action. However I haven't been able to get it to work yet. I need to configure personal access token, and it is possible that you may need to do some of the permission giving.

I'll find out what to do and let you know if needed.

```
 ! [remote rejected] unit-tests-and-github-actions -> unit-tests-and-github-actions (refusing to allow a Personal Access Token to create or update workflow `.github/workflows/test.yml` without `workflow` scope)
```